exercises/practice/perfect-numbers/.approaches/if-statements/content.md

```diff
@@ -18,14 +18,13 @@ public class PerfectNumbers
         if (number < 1)
             throw new ArgumentOutOfRangeException(nameof(number));
 
-        var sumOfFactors = Enumerable.Range(1, number / 2)
-            .Where(factor => number % factor == 0)
-            .Sum();
+        var sum = Enumerable.Range(1, number / 2)
```

Comment:
I quite like `sumOfFactors` better to be honest. It better reflects the domain IMHO.

Reply:
Agreed. I wanted to align this with the other approach, but I'll submit a separate PR to fix this there as well.